docs/resources/samples/external-fetch-calls.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Use external fetch calls in Office Scripts
 description: Learn how to make external API calls in Office Scripts.
-ms.date: 05/14/2021
+ms.date: 06/10/2022
 ms.localizationpriority: medium
 ---
 
 # Use external fetch calls in Office Scripts
 
-This script gets basic information about a user's GitHub repositories. It shows how to use `fetch` in a simple scenario. For more information about using `fetch` or other external calls, read [External API call support in Office Scripts](../../develop/external-calls.md)
+This script gets basic information about a user's GitHub repositories. It shows how to use `fetch` in a simple scenario. For more information about using `fetch` or other external calls, read [External API call support in Office Scripts](../../develop/external-calls.md). For information about working with [JSON]](https://www.w3schools.com/whatis/whatis_json.asp) objects, like what is returned by the GitHub APIs, read [Use JSON to pass data to and from Office Scripts](../../develop/use-json.md).
 
 You can learn more about the GItHub APIs being used in the [GitHub API reference](https://docs.github.com/rest/reference/repos#list-repositories-for-a-user). You can also see the raw API call output by visiting `https://api.github.com/users/{USERNAME}/repos` in a web browser (be sure to replace the {USERNAME} placeholder with your GitHub ID).
```

Comment:
```suggestion
Learn more about the GItHub APIs being used in the [GitHub API reference](https://docs.github.com/rest/reference/repos#list-repositories-for-a-user). You can also see the raw API call output by visiting `https://api.github.com/users/{USERNAME}/repos` in a web browser (be sure to replace the {USERNAME} placeholder with your GitHub ID).
```
The two "you cans" in a row jumped out at me!

Reply:
If the Yucatan toucan can use two "you cans", you can too!